### PR TITLE
[zh-cn]: Update translation of Object.__lookupGetter__()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

None.

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/object/__lookupgetter__/index.md [hunk 1]
```diff
@@ -3,73 +3,105 @@ title: Object.prototype.__lookupGetter__()
...
 
-不属于任何规范。
+{{Specifications}}
 
 ## 浏览器兼容性
 
 {{Compat}}
 
-## 相关链接
+## 常见
```
Comment (skyclouds2001) [inline files/zh-cn/web/javascript/reference/global_objects/object/__lookupgetter__/index.md:99]:
```suggestion
## 参见
```

File: files/zh-cn/web/javascript/reference/global_objects/object/__lookupgetter__/index.md [hunk 2]
```diff
@@ -3,73 +3,105 @@ title: Object.prototype.__lookupGetter__()
 slug: Web/JavaScript/Reference/Global_Objects/Object/__lookupGetter__
 ---
 
-{{JSRef("Global_Objects", "Object")}} {{ Non-standard_header }} {{ Deprecated_header }}
+{{JSRef}} {{deprecated_header}}
 
 ## 概述
 
-**`__lookupGetter__`** 方法会返回当前对象上指定属性的**属性读取访问器函数（getter）。**
+**备注：** 该特性已被弃用，建议使用 {{jsxref("Object.getOwnPropertyDescriptor()")}} API。该方法的行为仅在 Web 兼容性方面进行了规定，并不要求在任何平台上实现。它可能无法在所有地方正常工作。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/__lookupgetter__/index.md:10]:
```suggestion
> **备注：** 该特性已被弃用，建议使用 {{jsxref("Object.getOwnPropertyDescriptor()")}} API。该方法的行为只是出于 web 兼容性进行了规定，在任何平台上都不要求实现该方法。它可能无法在所有地方正常工作。
```

File: files/zh-cn/web/javascript/reference/global_objects/object/__lookupgetter__/index.md [hunk 3]
```diff
@@ -3,73 +3,105 @@ title: Object.prototype.__lookupGetter__()
...
+
+### 返回值
+
+绑定到指定属性的 getter 函数。如果找不到这样的属性或者该属性是[数据属性](/zh-CN/docs/Web/JavaScript/Data_structures#数据属性)，则返回 `undefined`。
+
+## 描述
+
+所有继承自 `Object.prototype` 的对象（也就是除了 [`null` 原型对象](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object#null_原型对象)的对象）都会继承 `__lookupGetter__()` 方法。如果为对象的属性定义了一个 [getter](/zh_CN/docs/Web/JavaScript/Reference/Functions/get)，则无法通过该属性引用 getter 函数，因为该属性引用该函数的返回值。`__lookupGetter__()` 可用于获取对 getter 函数的引用。
+
+`__lookupGetter__()` 沿着[原型链](/zh-CN/docs/Web/JavaScript/Inheritance_and_the_prototype_chain)向上查找指定的属性。如果原型链上的任何对象具有指定的[自有属性](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn)，则返回该属性的[属性描述符](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyDescriptor)的 `get` 属性。如果该属性是数据属性，则返回 `undefined`。如果在整个原型链上都找不到该属性，则也返回 `undefined`。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/__lookupgetter__/index.md:33]:
```suggestion
`__lookupGetter__()` 沿着[原型链](/zh-CN/docs/Web/JavaScript/Inheritance_and_the_prototype_chain)向上查找指定的属性。如果原型链上的任何对象具有指定的[自有属性](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn)，则返回该属性的[属性描述符](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyDescriptor)的 `get` 属性。如果该属性是数据属性，则返回 `undefined`。如果在整个原型链上都找不到该属性，也同样返回 `undefined`。
```

File: files/zh-cn/web/javascript/reference/global_objects/object/__lookupgetter__/index.md [hunk 4]
```diff
@@ -3,73 +3,105 @@ title: Object.prototype.__lookupGetter__()
...
+
+`__lookupGetter__()` 在规范中被定义为 "规范可选"，这意味着不需要实现此功能。但是，所有主要的浏览器都实现了它，并且由于其持续使用，它不太可能被移除。如果浏览器实现了 `__lookupGetter__()`，它还需要实现 [`__lookupSetter__()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/__lookupSetter__)、[`__defineGetter__()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/__defineGetter__) 和 [`__defineSetter__()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/__defineSetter__) 方法。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/__lookupgetter__/index.md:35]:
```suggestion
`__lookupGetter__()` 在规范中被定义为“规范可选”，这意味着不需要实现此特性。但是，所有主要的浏览器都实现了它，并且由于其持续使用，它不太可能被移除。如果浏览器实现了 `__lookupGetter__()`，它还需要实现 [`__lookupSetter__()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/__lookupSetter__)、[`__defineGetter__()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/__defineGetter__) 和 [`__defineSetter__()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/__defineSetter__) 方法。
```

File: files/zh-cn/web/javascript/reference/global_objects/object/__lookupgetter__/index.md [hunk 5]
```diff
@@ -3,73 +3,105 @@ title: Object.prototype.__lookupGetter__()
...
 
-obj.__lookupGetter__("foo")
-// (function (){return Math.random() > 0.5 ? "foo" : "bar"})
+obj.__lookupGetter__("foo");
+// [Function: get foo]
 ```
 
+### 以标准方式查找属性的 getter
+
 ## 附注
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/__lookupgetter__/index.md:55]:
```suggestion
```

**Review by yin1999 — COMMENTED** [review]
Some nits/suggestions. Otherwise looks good :+1:

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**